packages/backend/Dockerfile

```diff
@@ -19,15 +19,17 @@ RUN apt-get update && \
     yarn config set python /usr/bin/python3
 
 # From here on we use the least-privileged `node` user to run the backend.
-USER node
 WORKDIR /app
+RUN chown node:node /app
+USER node
```

Comment:
What's the benefit/goal of moving this? What are you trying to accomplish? 

FYI usually the mission statement is summarized in the PR title and greater detail is providing in the opening comment.

Reply:
It’s in the self-review if you see my comments

Reply:
This isn't an either-or situation though. `Update Dockerfile` is a pretty meaningless PR title. Anyone can see from you editing the file that it has been updated. What is your end goal though? What does updating it fix/improve/accomplish? You can edit the title and opening comment to make them both more meaningful.